src/telemetry.ts

```diff
@@ -1,30 +1,46 @@
-import appInsights from "./appInsights.js";
+import child_process from 'child_process';
 import { Cli } from "./cli/Cli.js";
 import { settingsNames } from "./settingsNames.js";
+import path from 'path';
+import { pid } from './utils/pid.js';
+import { session } from './utils/session.js';
 
-class Telemetry {
-  public trackEvent(commandName: string, properties: any): void {
+function trackTelemetry(object: any): void {
```

Comment:
I don't understand how this change is relevant to the PR. Could you please elaborate?